buildo/react-components
# #258: Meter (closes #258)

Issue #258

## Test Plan

- run `npm start` and check the examples
- i provide examples for every use case
- Open console and you will see 2 warning, in order to cover from some bad behaviour
- unit tests in #275
